### PR TITLE
changelog for 1.2.3

### PR DESCRIPTION
**Pull request by paulhainsworth-chia**

### CODE REVIEW

**Review by hoffmang9 — DISMISSED** [review]
looks good

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,42 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+## 1.2.3 Chia blockchain 2021-07-21
+
+### Added
+
+- Added ability to change payout instructions in the GUI.
+- Added an option to revert to sequential read. There are some systems (primarily macos+exfat) where the parallel read features results in very long lookup times. This addition makes the parallel feature the default, but adds the ability to disable it and revert back to sequential reads.
+- Added backwards compatibility for Coin Solutions in push_tx since renaming it to CoinSpend.
+- Added an option to set the default constants on the simulator.
+- Added a warning to user to not send money to the pool contract address.
+- Added new CLI command: 'chia plotnft show' to show plot count
```
Comment (emlowe) [inline CHANGELOG.md:18]: This command already existed - the output was expanded to include the plot count. It was also changed to output more information about pool points.

Comment (emlowe) [reply CHANGELOG.md:18]: Oh, and this last one is thanks to @mishan

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,42 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+### Changed
+
+- We dropped the 0x prefix from JSON conversions that will change results in both CLI and RPCs.
+- Updated blspy to 1.0.5.
+- Included all Chialisp files in source distribution.
```
Comment (emlowe) [inline CHANGELOG.md:26]: We also updated chiapos to 1.0.4

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,42 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Removed left-over debug logging from test_wallet_pool_store.
+- Made changes to allow us to use the name coin_spend everywhere in our code, without changing it in the API requests, both outgoing and incoming. Enables us to decide at a later date when to cut over completely to the coin_spend name.
+- Thanks @mishan for your change to the GUI to display Percent Successful Points, from previously relying on 'get_pool_state,' to updating the output to show using 'chia plotnft'.
```
Comment (emlowe) [inline CHANGELOG.md:29]: oh, I see you have this here. It was **not** a change to the GUI - was a change to chia plotnft show.

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,42 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+### Fixed
+
+- Thanks to @dfaranha for helping fix a parsing error in Relic inputs for BLS signatures
+- Fixed error type in wallet_blockchain.py
+- Thanks to @seraphik for a fix on our Linux installer that required root privileges.
+- Thanks @felixbrucker for helping fix invalid content-type header issues in pool API requests.
+
```
Comment (emlowe) [inline CHANGELOG.md:37]: New fixed: The wallet ignores coins sent by accident to the pool contract address and allows self pooling rewards to be claimed in this case.

Comment (paulhainsworth-chia) [reply CHANGELOG.md:37]: Thanks @emlowe I'll update changelog to include this fix.

File: CHANGELOG.md (hunk 3 above)
Comment (paulhainsworth-chia) [reply CHANGELOG.md:29]: Fixed, thanks.

File: CHANGELOG.md (hunk 1 above)
Comment (mishan) [reply CHANGELOG.md:18]: This command already existed. I just added an output line that displays the % of successful points over the last 24 hours

Comment (mariano54) [reply CHANGELOG.md:18]: @mishan added the % succesful points, we should capture that in the changelog @paulhainsworth-chia

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,42 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+### Changed
+
+- We dropped the 0x prefix from JSON conversions that will change results in both CLI and RPCs.
```
Comment (mariano54) [inline CHANGELOG.md:24]: We will probably revert this (bills PR)

File: CHANGELOG.md (hunk 3 above)
Comment (mariano54) [reply CHANGELOG.md:29]: change to the CLI not GUI

File: CHANGELOG.md (hunk 5 above)
Comment (wjblanke) [reply CHANGELOG.md:24]: ya its been reverted

**Comment by AlexSSD7** [comment]
Please, don't forget to include PR https://github.com/Chia-Network/chia-blockchain/pull/7731 in the upcoming 1.2.3 release. This is a serious bug, and many Flexpool farmers are complaining because they are getting underpaid.

cc @hoffmang9 @mariano54 (apologies for CC'ing)

**Review by wjblanke — APPROVED** [review]
aok

**Pull request merged**